homeassistant/util/unit_system.py

```diff
@@ -189,24 +194,42 @@ def as_dict(self) -> dict[str, str]:
         }
 
 
-METRIC_SYSTEM = UnitSystem(
-    CONF_UNIT_SYSTEM_METRIC,
-    TEMP_CELSIUS,
-    LENGTH_KILOMETERS,
-    SPEED_METERS_PER_SECOND,
-    VOLUME_LITERS,
-    MASS_GRAMS,
-    PRESSURE_PA,
-    LENGTH_MILLIMETERS,
-)
+class _MetricUnitSystem(UnitSystem):
+    def __init__(self, name: str) -> None:
+        super().__init__(
+            name,
+            TEMP_CELSIUS,
+            LENGTH_KILOMETERS,
+            SPEED_METERS_PER_SECOND,
+            VOLUME_LITERS,
+            MASS_GRAMS,
+            PRESSURE_PA,
+            LENGTH_MILLIMETERS,
+        )
 
-IMPERIAL_SYSTEM = UnitSystem(
-    CONF_UNIT_SYSTEM_IMPERIAL,
-    TEMP_FAHRENHEIT,
-    LENGTH_MILES,
-    SPEED_MILES_PER_HOUR,
-    VOLUME_GALLONS,
-    MASS_POUNDS,
-    PRESSURE_PSI,
-    LENGTH_INCHES,
-)
+    @property
+    def is_metric(self) -> bool:
+        return True
```

Comment:
I think my previous comment is lost in translation or misunderstood.

My point was, we don't need this property at all?

The fact the unit system is an instance of `MetricUnitSystem` should already provide this, right?